stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -2983,12 +2986,12 @@ func.func @dot_general(%arg0: tensor<?x?x?xf32>, %arg1: tensor<?x?x?xf32>) {
       rhs_contracting_dimensions = [1]
     >
   } : (tensor<?x?x?xf32>, tensor<?x?x?xf32>) -> tensor<?x?x?xf32>
-  func.return
+  func.return %0 : tensor<?x?x?xf32>
 }
 
 // -----
 
-func.func @dot_general(%arg0: tensor<*xf32>, %arg1: tensor<*xf32>) {
+func.func @dot_general_c1(%arg0: tensor<*xf32>, %arg1: tensor<*xf32>) -> tensor<*xf32>{
```

Comment:
Do we need two tests for C1? (Same for other constraints below, e.g. C2 and C3).

Reply:
No, these should be removed according to the latest checklist.